boxes/identity-server/provisioner/provisioner.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------
+# Copyright 2017 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ------------------------------------------------------------------------
+
+# This script acts as the machine provisioner during the Vagrant box build process for WSO2 Identity Server.
+
+# set variables
+WSO2_SERVER=wso2is
+WSO2_SERVER_VERSION=5.4.0
+WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}.zip
+JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
+MYSQL_CONNECTOR=mysql-connector-java-*-bin.jar
+
+DEFAULT_MOUNT=/vagrant
+SOFTWARE_DISTRIBUTIONS=${DEFAULT_MOUNT}/files
+CONFIGURATIONS=${DEFAULT_MOUNT}/identity-server/confs
+WORKING_DIRECTORY=/home/vagrant
+JAVA_HOME=/opt/java
+DEFAULT_USER=vagrant
+
+# operate in anti-fronted mode with no user interaction
+export DEBIAN_FRONTEND=noninteractive
+
+# check if the required software distributions have been added
+if [ ! -f ${SOFTWARE_DISTRIBUTIONS}/${WSO2_SERVER_PACK} ]; then
+    echo "WSO2 server pack not found"
```

Comment:
Please implement the same suggestion given in `identity-server-analytics` provisioner section.

Reply:
+1